terraform/modules/configs/deploy-all/main.tf

```diff
@@ -405,6 +420,14 @@ resource "aws_iam_role_policy_attachment" "ecs_task_execution_role_read_ssm" {
 # Modules
 ######################################
 
+module "s3_virus_scan_lambda" {
```

Comment:
Although this will work, it's not quite what I meant.  Instead of moving the source _and_ declaring a separate module for the lambda, just move the source (as you've done) and incorporate it into the existing `module "s3_virus_scan"` as it is part and parcel of that 'service' (probably as a child module).  
Two modules sharing the  same source could get confusing otherwise..